test/rescue_test.rb

```diff
@@ -15,125 +15,99 @@ def execute(*args)
       end
 
       let :rescued_plan do
-        execution_plan.rescue_from_error.value
+        world.persistence.load_execution_plan(execution_plan.id)
```

Comment:
Instead of calling the method that is being used only in tests, we look at the current state of execution plan after the execution finished